### PR TITLE
Detailpage

### PR DESCRIPTION
**Pull request by JannikUekoetter**
Here a longer PR was created, because I had some problems with the implementation of the dummy data on the detail page.

### CODE REVIEW

File: components/Db.js
Comment (PhilippWeiss85) [inline components/Db.js:160]: if you don't want to edit every id manually you could just install nanoid for random unique ids.. super easy to use (did it myself and can recommend it) 
https://github.com/ai/nanoid

File: components/FoundItem.js
Comment (PhilippWeiss85) [inline components/FoundItem.js:9]: I am not quite sure if this is the right spot for the <Link />. Because now you could click like everywhere within the fragment and get to the details page. I would suggest to put the Link around the Image (would make more sense to me at the moment)

File: components/LostItem.js
Comment (PhilippWeiss85) [inline components/LostItem.js:8]: Same feedback as above. Please check your link should really wrap the whole content.

File: pages/contact.js
Comment (PhilippWeiss85) [inline pages/contact.js]: Contact() with an uppercase C

File: pages/details/[id].js
Comment (PhilippWeiss85) [inline pages/details/[id].js:6]: Not sure but I think this should be an async function

**Review by PhilippWeiss85 — CHANGES_REQUESTED** [review]
Hey Jannik, please see comments. Maybe a second review for the SSR could be benefical

File: components/Db.js
Comment (JannikUekoetter) [reply components/Db.js:160]: For the future i will definitely rethink this. Currently the dummy data remains and will not be greatly expanded for the time being. Thanks for the good idea!

File: pages/contact.js
Comment (JannikUekoetter) [reply pages/contact.js]: I have deleted the contact page in the pages because it is not currently used and this causes problems with vercel.

File: pages/details/[id].js
Comment (JannikUekoetter) [reply pages/details/[id].js:6]: Here I already had Felix look over it and I think it doesn't have to be async since the data is pulled from db.js.

**Review by PhilippWeiss85 — APPROVED** [review]
You got my approval mate! However if I were you I would still wait for another review. :)

**Pull request merged**
